lib/Db/FolderMapperV2.php

```diff
@@ -101,14 +101,14 @@ public function findFromUser(string $userId, int $id): Entity
     /**
      * @param string   $userId
      * @param int      $id
-     * @param int|null $maxItemID
```

Comment:
The new naming is clearer to me so I'd rather keep it here in the mapper.

Reply:
Ok then we have to live with a bit of inconsistency through the stack but that is ok for me, I will revert that part.
Then I also don't need to adjust the test.